source/2019-07-26-the-ember-times-issue-108.md

```diff
@@ -23,12 +23,12 @@ learn how to use Sentry to track production errors 📈,
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<consider adding some bold to your paragraph>
+## [How to Hire and Train Developers when You Don’t Use React 📚](https://medium.com/front-end-weekly/how-to-hire-and-train-developers-when-you-dont-use-react-42762e6b1a57)
   
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+[@jenweber](https://github.com/jenweber) wrote an awesome blogpost on how to hire and train new developers.
+In 10 steps she points out the pain points of every new hire and how to solve them as a company.
+If you are in charge of hiring new people or if you have a new colleague in your team, this is definitively a nice piece to get some new ideas about your onboarding process.
+When you are the lucky one to have started a new job, this post is a nice read to realize that you are not the only one with start-up problems and it might help you formulate what you are missing right now.
```

Comment:
The second part of this sentence is a little confusing. I think I get what you meant to convey, though. I think you wanted to inspire new hires to be thankful of their mentors and be proactive in their training?

Reply:
Yeah, and I wanted to inspire people who just started to read the blogpost because at least in my opinion it also showcases really well what you could expect and also contribute when you start working somewhere else

Reply:
Hmm, how about then? Feel free to change my suggestion as you see fit.

```suggestion
If you have started a new job, this post is also for you! We encourage you to think about how you want to be trained and how you can contribute to your team's onboarding process.
```

Reply:
thank you both for the work on this 💯